events/carousel/specific_carousel_slide.md

```diff
@@ -7,8 +7,8 @@ Fire whenever a user interacts with the specific carousel slide control
 ```html
 <a href="<link_url>"
   data-layer-event="specific_carousel_slide"
-  data-layer-carousel_id="<carousel_id>"
-  data-layer-carousel_name="<carousel_name>"
+  data-layer-event_specific_id="<event_specific_id>"
+  data-layer-event_specific_id="<event_specific_id>"
```

Comment:
did you mean to have `event_specific_name` here instead of `event_specific_id` twice?

Reply:
Yes